[2024-05-13T17:03:19.146+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: live_ademe.save_postgresdb manual__2024-05-13T16:58:11.467547+00:00 [queued]>
[2024-05-13T17:03:19.157+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: live_ademe.save_postgresdb manual__2024-05-13T16:58:11.467547+00:00 [queued]>
[2024-05-13T17:03:19.158+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 2
[2024-05-13T17:03:19.176+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): save_postgresdb> on 2024-05-13 16:58:11.467547+00:00
[2024-05-13T17:03:19.187+0000] {standard_task_runner.py:60} INFO - Started process 24166 to run task
[2024-05-13T17:03:19.194+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'live_ademe', 'save_postgresdb', 'manual__2024-05-13T16:58:11.467547+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/load.py', '--cfg-path', '/tmp/tmpap64oi4c']
[2024-05-13T17:03:19.198+0000] {standard_task_runner.py:88} INFO - Job 95: Subtask save_postgresdb
[2024-05-13T17:03:19.318+0000] {task_command.py:423} INFO - Running <TaskInstance: live_ademe.save_postgresdb manual__2024-05-13T16:58:11.467547+00:00 [running]> on host 86724742ac03
[2024-05-13T17:03:19.546+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='live_ademe' AIRFLOW_CTX_TASK_ID='save_postgresdb' AIRFLOW_CTX_EXECUTION_DATE='2024-05-13T16:58:11.467547+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-13T16:58:11.467547+00:00'
[2024-05-13T17:03:19.549+0000] {load.py:133} INFO - openning result file
[2024-05-13T17:03:19.577+0000] {load.py:138} INFO - dataframe creation
[2024-05-13T17:03:19.600+0000] {load.py:142} INFO - dataframe creation
[2024-05-13T17:03:19.636+0000] {load.py:146} INFO - dataframe cleaning
[2024-05-13T17:03:19.697+0000] {load.py:151} INFO - dataframe storing
[2024-05-13T17:03:19.700+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/load.py", line 152, in save_postgresdb
    data[["n_dpe", "payload"]].to_sql(
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/frame.py", line 3767, in __getitem__
    indexer = self.columns._get_indexer_strict(key, "columns")[1]
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 5877, in _get_indexer_strict
    self._raise_if_missing(keyarr, indexer, axis_name)
  File "/home/airflow/.local/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 5941, in _raise_if_missing
    raise KeyError(f"{not_found} not in index")
KeyError: "['payload'] not in index"
[2024-05-13T17:03:19.722+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=live_ademe, task_id=save_postgresdb, execution_date=20240513T165811, start_date=20240513T170319, end_date=20240513T170319
[2024-05-13T17:03:20.203+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 95 for task save_postgresdb ("['payload'] not in index"; 24166)
[2024-05-13T17:03:20.231+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-05-13T17:03:20.266+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
